app/Tasks/Start.php

```diff
@@ -73,16 +90,34 @@ public function checkDependencies()
                 return 1;
             }
 
-            return $this->runCallableWaitFor(function () {
-                return $this->runCommandWithoutOutput(
-                    DockerCompose::make('ps'),
-                    false
-                );
-            }, $this->timeout);
+            return 0;
+        });
+    }
+
+    public function handleNetworkTask() : int
+    {
+        return $this->runTask('Setting up network', function () {
+            return $this->handleNetwork();
         });
     }
 
-    public function dockerComposeUpD()
+    public function handleNetwork() : int
+    {
+        $command = Docker::make('network', 'ls', '-f', 'NAME=' . env('FWD_NETWORK'));
```

Comment:
using `-q` simplifies the logic a bit:

![image](https://user-images.githubusercontent.com/347400/71531032-9adf0280-28cb-11ea-8b17-449609273e86.png)